### PR TITLE
test that multiple protocols mapped to the same VIP_0 parse properly

### PR DESCRIPTION
**Pull request by jdef**

### CODE REVIEW

**Review by nlsun — APPROVED** [review]

File: src/minuteman_mesos_poller.erl [hunk 1]
```diff
@@ -585,6 +585,26 @@ state4_test() ->
...
 
+state5_test() ->
+    {ok, Data} = file:read_file("testdata/state5.json"),
+    {ok, MesosState} = mesos_state_client:parse_response(Data),
+    VIPBes = collect_vips(MesosState, fake_state()),
+    Expected = [
+        {
+            {tcp, {1, 2, 3, 4}, 5000},
+            [
+                {{10, 0, 0, 243}, {{10, 0, 0, 243}, 26645}}
```
Comment (GoelDeepak) [inline src/minuteman_mesos_poller.erl]: This format has changed in master to {{10, 0, 0, 243}, 26645}

File: src/minuteman_mesos_poller.erl [hunk 2]
```diff
@@ -585,6 +585,26 @@ state4_test() ->
...
+            ]
+        },
+        {
+            {udp, {1, 2, 3, 4}, 5000},
+            [
+                {{10, 0, 0, 243}, {{10, 0, 0, 243}, 26645}}
```
Comment (GoelDeepak) [inline src/minuteman_mesos_poller.erl]: Please update this as well.

**Comment by jdef** [comment]
done

**Comment by GoelDeepak** [comment]
lgtm

**Review by aeyakovenko — APPROVED** [review]

**Pull request merged**